R/build_taxonomy.R

```diff
@@ -7,6 +7,10 @@
 #' @return Data frame with the taxonomic information.
 #' @noRd
 build_taxonomy <- function(x) {
+
+  # Check Camtrap DP object
+  check_camtrapdp(x)
```

Comment:
This is a private function, called from `read_camtrapdp()`. cf. `convert()`, I don't think checking is necessary.